modules/apps/document-library/document-library-web/.npmbundlerrc

```diff
@@ -18,12 +18,6 @@
 				"clay-portal": ">=2.9.0",
 				"clay-radio": ">=2.9.0"
 			},
```

Comment:
I think in this case all of `frontend-taglib-clay` could be removed as well since there are no references to it neither in `package.json` nor in the code in the module.

I know it might feel like some times I ask you to do the extra work and others I ask you not to... hope there's a difference between both cases. In here, this seems like useless configuration that has very minimal chance of affecting the module's behaviour... 🤞

Reply:
Yep, I get it @jbalsas. 👍 

Should I do this more often? By `this` i mean when touching files, to check if there are some redundancies, like in this case `frontend-taglib-clay` that could be removed. Only in very simple cases like this one, I assume.

Reply:
> Should I do this more often?

Attention to detail is a good thing, so it's good to check and double-check things as you go. 👍

Having said that, there are lots of "errors" that machines are very good at catching and should be automated. This might be one of those things.

Reply:
> > Should I do this more often?
>
> Attention to detail is a good thing, so it's good to check and double-check things as you go. 👍

There's also the whole [Systems Thinking for Developers](https://vimeo.com/showcase/8258302/video/520133397). Sometimes you don't have to zoom out so much to make these connections. Just looking around a little bit, understand the system you're interacting with and get a sense of things out of place.